src/jagruti/Assignment11_JagrutiM.java

```diff
@@ -0,0 +1,25 @@
+package jagruti;
+
+public class Assignment11_JagrutiM
+{
+	//a method to compute simple interest   
+	void CalculateSimpleInterest(int P,int T,int R)
```

Comment:
method name should be in small & variable too

Reply:
![Assignment11_Jagruti](https://user-images.githubusercontent.com/112231251/187497859-7c7b348b-1f7b-4991-b968-73d2f5a26f7c.png)